app/Http/Controllers/AdminConabController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Facades\Hash;
+
+class AdminConabController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $user = Auth::user();
+        $users = User::where('user_type', 'ADMIN_CONAB')->where('id', '<>', $user->id)->get();
+        return $users;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $data = Validator::make($request->all(), [
+            'name' => 'required|string',
+            'email' => 'required|email',
+            'cpf' => 'required|regex:/^[0-9]{3}\.[0-9]{3}\.[0-9]{3}\-[0-9]{2}/',
+            'phones' => 'required',
```

Comment:
O telefone também deve ser único segindo as migrações, dentro do contexto dos admins acho isto correto, mas para os consumidores acho um problema, pois pode haver telefone compartilhado dentro da família, o filho cadastra um telefone e a mãe cadastra o mesmo, aplicação vai negar o cadastro do telefone, isso ta certo?

Reply:
É se for pela migração vai dá pau no banco quando for registrar o mesmo telefone no caso desse exemplo ae. Mas vou alterar também para os administradores.

Reply:
Você pode optar pelo `distinct` para fazer essa validação.